src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -12,6 +12,7 @@ CA1842 | Performance | Info | DoNotUseWhenAllOrWaitAllWithSingleArgument, [Docum
 CA1843 | Performance | Info | DoNotUseWhenAllOrWaitAllWithSingleArgument, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1843)
 CA1844 | Performance | Info | ProvideStreamMemoryBasedAsyncOverrides, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1844)
 CA1845 | Performance | Info | UseSpanBasedStringConcat, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1845)
+CA2250 | Usage | Warning | DetectPreviewFeatureAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2250)
```

Comment:
Did we decide on Warning, or were we still considering making this an Error by default?

Reply:
We should make this error by default I think. @terrajobst

Reply:
Analyzers should never report an error by default. Error severity is technically reserved for compiler errors and extremely critical security analyzers.

Tagging @sharwell as well.

Reply:
This scenario might justify an exception to that rule since we need users to take explicit action to enable preview features if they want to use them. Not everyone has warnings as errors enabled, and this message could be lost in a sea of other warnings. This logic will be implemented by the compiler for F# too. The fact that this is an analyzer is a factoring/implementation choice--not a key aspect of the user experience, and we're leaving the door open that this could get "promoted" into the compiler once the feature and its details are well proven.

Reply:
> Analyzers should never report an error by default. Error severity is technically reserved for compiler errors and extremely critical security analyzers.

Is this a guideline or a rule? In either case, do you know why analyzers prefer warnings vs errors?

Reply:
Here's some background discussion that might also be helpful:
https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-05-12.md#discussion

Reply:
@pgovind It’s a guideline. We normally don’t want analyzer diagnostics to prevent building the assembly, which blocks users from doing operations like Run Tests, Start Debugging, etc.

@jeffhandley Thanks for the context. It seems clear that this is a very special analyzer, and analyzer implementation is an interim step towards making it a compiler feature. So, this deserves an exception. I’ll let you all go ahead with warning vs error choice.